Resources/Prototypes/Entities/Structures/Wallmounts/WallmountMachines/Monitors/telescreens.yml

```
@@ -12,6 +12,17 @@
         state: telescreen_frame
       - map: ["computerLayerScreen"]
         state: telescreen
+  - type: Destructible
+    thresholds:
+    - trigger:
+      !type:DamageTrigger
+      damage: 200
```

```suggestion
        !type:DamageTrigger
        damage: 200
```
keep this indentation
same below